front-end/src/screens/SignupForm/index.tsx

```diff
@@ -133,6 +134,28 @@ const SignupForm = ({ className }:Props): JSX.Element => {
 							{errors.password && <span className={'errorText'}>{messages.VALIDATION_PASSWORD_ERROR}</span>}
 						</Form.Field>
 					</Form.Group >
+					<Form.Field>
+						<input
+							className={errors.termsandconditions ? 'error' : ''}
+							name='termsandconditions'
+							value='yes'
+							ref={register({ required: true })}
+							type='checkbox'
+						/>
+						<div className='text-muted'>I agree to the <Link to='/terms-and-conditions'>Terms and Conditions</Link></div>
```

Comment:
Can we use html5 label here and make sure that clicking on the text checks the box.
https://stackoverflow.com/questions/6293588/how-to-create-a-checkbox-with-a-clickable-label
That's a usability standard I think.